packages/es-dev-server/package.json

```diff
@@ -137,7 +137,7 @@
     "request": "^2.88.0",
     "selfsigned": "^1.10.4",
     "ts-mocha": "^7.0.0",
-    "typescript": "^3.8.3",
+    "typescript": "~3.8.3",
```

Comment:
we have typescript as a "root" dependency and it should not be repeated in devDependencies...

```suggestion

```